doc/rst/usingchapel/building.rst

```diff
@@ -62,132 +56,156 @@ display its help message using:
 
   chpl --help
 
-In which case, you will be ready to move on to compiling with the
-Chapel compiler (see :ref:`readme-compiling`).  The rest of this
-file gives more information about Chapel's Makefiles for advanced
+In which case, you are ready to move on to compiling with the Chapel
+compiler (see :ref:`readme-compiling`).  The next few sections provide
+more information about using Chapel across multiple sessions,
+installing Chapel, and supporting multiple Chapel configurations.
+Following that are sections describing Chapel's Makefiles for advanced
 users or developers of Chapel.
 
 
-----------------
-Platform Support
-----------------
+.. _using-chapel-in-other-sessions:
+
+------------------------------------------------------
+Using Chapel in a Different Shell / Terminal / Session
+------------------------------------------------------
+
+Once you have built Chapel, you can use it in other sessions (new
+shells or terminals) as long as:
+
+1. the ``chpl`` compiler is in your ``$PATH``
+
+#. your ``$CHPL_`` environment variables are set or inferred to values
+   that match a configuration you have already built
+
+Typically, this is accomplished in one of the following ways:
+
+* by manually re-sourcing the ``setchplenv.*`` file that you used
+  when building Chapel.
 
-Currently supported platforms include 32- and 64-bit Linux, Mac OS X,
-Cygwin (Windows), SunOS, a variety of current HPE and Cray platforms, and a
-few systems by other vendors.  Most UNIX-based environments ought to
-support Chapel (subject to the assumptions in :ref:`readme-prereqs`), but may
-not be supported "out-of-the-box" by our current Makefile structure.
-See the section below on platform-specific settings for more
-information on adding support for additional UNIX-compatible
-environments.
+* by editing your 'dot files' (e.g., ``.bashrc``, ``bash_profile``, or
+  whatever your shell uses to start up each session) to explicitly set
+  your path and ``$CHPL_`` settings as desired, or to source the
+  appropriate ``setchplenv.*`` script.
 
-Note that a single Chapel installation can simultaneously support
-Chapel for multiple platforms and compiler options because all
-platform-specific binary files and executables are stored in
-subdirectories named by ``CHPL_`` environment variables.
+* by creating a ``chplconfig`` file to store your preferred ``$CHPL_``
+  settings (see :ref:`readme-chplenv.chplconfig` for further details
+  on creating a chplconfig file).  Note, however, that some other
+  standard technique is still required to ensure that ``chpl`` is in
+  your path.
+
+* by installing Chapel to a specific location—perhaps one that is
+  already in your path—using the instructions in the next section.
+
+
+.. _readme-installing:
+
+-----------------
+Installing Chapel
+-----------------
+
+Chapel can be built and installed to a specific location as follows:
+
+.. code-block:: bash
+
+  ./configure  # use ``./configure --help`` for specific options
```

Comment:
Take the backquotes out of the ./configure --help in the comment, maybe just use regular quotes?  It looks strange when viewing the file from https://github.com/chapel-lang/chapel/blob/f1395f93db429732b760b7d0ad7f9127d42f27aa/doc/rst/usingchapel/building.rst.

Reply:
Good catch!